jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -145,10 +146,19 @@ def _on_dataset_selected_changed(self, event={}):
         if self.config != "cubeviz":
             return
         # self.dataset might not exist when app is setting itself up.
-        if hasattr(self, "dataset") and self.dataset.selected_dc_item.ndim > 2:
-            self.is_cube = True
-        else:
-            self.is_cube = False
+        if hasattr(self, "dataset"):
+            if isinstance(self.dataset.selected_dc_item, list):
+                datasets = self.dataset.selected_dc_item
+            else:
+                datasets = [self.dataset.selected_dc_item]
+            for dataset in datasets:
+                # This assumes all cubes, or no cubes. If we allow photometry on collapsed cubes
+                # or images this will need to change.
+                if dataset.ndim > 2:
+                    self.is_cube = True
+                    break
+                else:
+                    self.is_cube = False

```

Comment:
This can be set outside of `for`?

Reply:
I personally prefer this to setting it to False first and then overriding after 🤷‍♂️

Reply:
In that case, `self.is_cube = False` runs every iteration (until loop ends or set to `True`) whether that assignment is need or not. But given all the other performance bottlenecks we have, this is probably nothing, so okay for me if you prefer this.

Reply:
I have no preference, it could be set first since it's `break`ing once set to `True`.

Reply:
oh I misunderstood, yes, the `else` should be indented out one level to be inline with the `for`